src/AxelarProxy.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import { AxelarExecutable } from "lib/axelar-gmp-sdk-solidity/contracts/executable/AxelarExecutable.sol";
+import { Address } from "@openzeppelin/contracts/utils/Address.sol";
+
+contract AxelarProxy is AxelarExecutable {
```

Comment:
What permissions exist for this proxy now that `Ownable` no longer applies?

Reply:
None, it is completely immutable

Reply:
Yep sounds good.